setup.cfg

```diff
@@ -1,7 +1,36 @@
-
-[bdist_wheel]
-universal = 1
-
 [metadata]
+name = appdirs
+author = Trent Mick
+author_email = trentm@gmail.com
+maintainer = Trent Mick; Sridhar Ratnakumar; Jeff Rouse
+maintainer_email = trentm@gmail.com; github@srid.name; jr@its.to
+license = MIT
 license_file = LICENSE.txt
+description = A small Python module for determining appropriate platform-specific dirs, e.g. a "user data dir".
+long_description = file: README.rst, CHANGES.rst
+keywords = application, directory, log, cache, user
+url = https://github.com/ActiveState/appdirs
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: PyPy
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development :: Libraries :: Python Modules
 
+setup_requires = setuptools>=30.3.0; wheel
```

Comment:
I just want to remind you about b68a72e5472ead00a7ff3772dee672e46a70ecdb that `setuptools` is actually not required.  Though, if `distutils` reads the `setup_requires` option or not, I am not exactly sure.  Forgive my if it does not.